dev-manuals/testdata/README.md

```diff
@@ -3,7 +3,7 @@
 ## Introduction
 
 Since we have a lot of separate databases (at least one for each backend), we need to have an easy way to initialize them with test data.
-For our [test plans](../test-plans/README.md) it is also important that different sets of test data can be loaded into the databases.
+For our [test plans](../test-plans/index.rst) it is also important that different sets of test data can be loaded into the databases.
```

Comment:
problem readthedocs vs github again :/

Reply:
Not ReadTheDocs vs. GitHub, I simply deleted the README because all information is now in the template, and I hate unnecessary duplication.

Reply:
```suggestion
For our [test plans](../test-plans) it is also important that different sets of test data can be loaded into the databases.
```
Maybe it is better to link to the directory of the testplans. The `index.rst` view is in my Opinion not helpful and to view the plans you also need to go into the directories

Reply:
This might be true for GitHub, but for ReadTheDocs, the `index.rst` is required, it won't work otherwise.